records/new_metadata/newmeta_spec.yaml

```diff
@@ -57,6 +57,23 @@ pub:
   type: pub
   edge_annotation: true
 
+upstream:
+  description: Number of postsynaptic densities on a neuron in a specific region. Use this to annotate has_presynaptic_terminal_in relationships.
+  type: int
+  edge_annotation: true
+
+downstream:
+  description: Number of downstream synaptic connections (PSDs postsynaptic to the neuron T-bars) on a neuron in some specified region.  Use this to annotate 'has_postynaptic_terminal_in' relationships.
```

Comment:
'Number of **annotated** downstream synaptic connections (number of PSDs postsynaptic to the neuron T-bars) on a neuron in **a specific** region.'?

Make annotated status explicit and change to 'in a specific' to match the description for 'upstream' above?

'Use this to annotate 'has_preynaptic_terminal_in' relationships.'

I think I'm misunderstanding these but should it not be pre here?